apps/mempool/docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+version: "3.7"
+
+x-logging:
+  &default-logging
+  driver: journald
+  options:
+    tag: "umbrel-app {{.Name}}"
+
+services:
+  app:
```

Comment:
Should be named `web` to keep consistency with other apps.

Reply:
Fixed [0c33e79332d6b33a218d8f4c450192a5e65a83be](https://github.com/getumbrel/umbrel/pull/470/commits/0c33e79332d6b33a218d8f4c450192a5e65a83be)